samples/graph-appcatalog-lifecycle/csharp/README.md

```diff
@@ -0,0 +1,89 @@
+﻿### AppCatalog sample
+
+Bot Framework v4 AppCatalog sample.
+
+This sample shows a AppCatalog bot and demonstrates teamsApp lifecycle in catalog followed by commands given to Bot.
+
+## Prerequisites
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+  B) Or from Visual Studio code
```

Comment:
Could you please correct the sequence of steps? This one started with B directly.

Reply:
Also correct the folder specficied in the step here: 

 **Navigate to `graph-appcatalog-lifecycle/node` folder**
 Expected: 
 **Navigate to `samples/graph-appcatalog-lifecycle/csharp` folder**